guides/common/modules/proc_enabling-the-installer-managed-dhcp-service.adoc

```diff
@@ -0,0 +1,66 @@
+[id="enabling-the-installer-managed-dhcp-service"]
+= Enabling the installer-managed DHCP service
+
+If you do not have a DHCP server available in your network, you can use the installer-managed DHCP service. This feature enables you to provide a DHCP service with a low maintenance effort.
+
+Perform the steps on the {Project} or {SmartProxyServer} that you want to configure to manage the DHCP service for the subnet.
+
+
+.Prerequisites
+
+* You know the following network information:
+** The range of IP addresses the DHCP should manage
+** The IP address of the default gateway in the subnet
+** The IP addresses of the name servers for the subnet
+
+
+.Procedure
+
+. Backup the answer files:
++
+[options="nowrap",subs="+quotes,attributes"]
+....
+# mkdir /root/backup-answer-files/
+# cp -p /etc/foreman-installer/scenarios.d/*-answers.yml /root/backup-answer-files/
+....
+
```

Comment:
I was a bit too brief in https://github.com/theforeman/foreman-documentation/pull/3530#discussion_r2048638507 and https://github.com/theforeman/foreman-documentation/pull/3530#discussion_r2048638200 but I think we should drop the backups. We don't do that anywhere else.
```suggestion
```

Reply:
removed